packages/bot-web-ui/src/components/load-modal/recent-footer.tsx

```diff
@@ -7,23 +7,23 @@ import RootStore from 'Stores/root-store';
 type TRecentFooterProps = {
     is_open_button_loading: boolean;
     loadFileFromRecent: () => void;
+    setOpenSettings: (toast_message: string, show_toast?: boolean) => void;
```

Comment:
this type declaration

```
(toast_message: string, show_toast?: boolean) => void;
```

is present in different files. It would be better if we can have a file for reusable types